russian/sleep/brushing/brushing.md

```diff
@@ -1,3 +1,6 @@
 Вы идете в ванную почистить зубы. В стаканчике лежит две зубные счетки,
 красная и синяя, и вы застываете в размышлении и задаете себе вопрос: 
-"Какую из зубных счеток выбрать?"
\ No newline at end of file
+"Какую из зубных счеток выбрать?"
+
+[Вы выбрали синюю щетку] (choice/blue-toothbrush.md)
+[Вы выбрали красную щетку] (choice/red-toothbrush.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.